Framework/API/src/Citation.cpp

```diff
@@ -0,0 +1,93 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+
+#include "MantidAPI/Citation.h"
+
+#include <nexus/NeXusFile.hpp>
+
+namespace Mantid {
+namespace API {
+
+/**
+  This function is designed for construction and validation for loading, however
```

Comment:
What is this comment referencing? The constructor?

Also what is `getCitation`?.

Reply:
Legacy comment, I shall clean it up.